Dictionaries/eng_OCRFixReplaceList.xml

```diff
@@ -2536,6 +2536,7 @@
     <Line from="ISIGHS]" to="[SIGHS]" />
   </WholeLines>
   <RegularExpressions>
+    <RegEx find="[bB]eyonce" replaceWith="Beyoncé" />

```

Comment:
I think this should be added to all other languages...

Reply:
I think Beyonce / Beyoncé are both correct... so should be removed even though it will probably be correct 90% of the times.